components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java

```diff
@@ -347,7 +347,7 @@ public List selectAppOverallMetrics(String metricsType, String workerId, long
         switch (metricsType) {
```

Comment:
Better to move these to a query config file than hardcoding here.

Reply:
Fixed in 84eed0f2066c8da5780a1ab6e5bbed953ff210b9